tests/rptest/tests/rpk_start_test.py

```diff
@@ -41,10 +42,37 @@ def test_simple_start(self):
             "WARNING: This is a setup for development purposes only")
 
     @cluster(num_nodes=3)
-    def test_simple_start_three(self):
+    def test_simple_start_three_with_seeds(self):
         """
         Validate simple start using rpk with multiple nodes. Node IDs should be
-        assigned automatically by Redpanda.
+        assigned automatically by Redpanda, configuring seeds-driven cluster
+        formation.
+        """
+        node_ids = set()
+        seeds_str = ",".join(
+            [f"{n.account.hostname}" for n in self.redpanda.nodes])
+
+        def start_with_rpk(node):
+            seeds_arg = f"--seeds={seeds_str}"
+            bootstrap_arg = "--set redpanda.empty_seed_starts_cluster=false"
```

Comment:
These args go to `rpk redpanda start` don't they? I can't see an option called `--set` in that command: https://github.com/redpanda-data/redpanda/blob/dev/src/go/rpk/pkg/cli/cmd/redpanda/start.go#L119-L132

Reply:
It's a cobra (flags library) argument, e.g. see the comment here https://github.com/redpanda-data/redpanda/blob/dev/src/go/rpk/pkg/cli/cmd/redpanda/start.go#L146